js/ui/draggable.js

```diff
@@ -791,29 +797,31 @@ var Draggable = DOMComponentWithTemplate.inherit({
 
     _updateScrollable: function(e) {
         var that = this,
-            $dragElement = that._$dragElement,
-            ownerDocument = $dragElement.get(0).ownerDocument,
             $window = $(window),
             mousePosition = {
                 x: e.pageX - $window.scrollLeft(),
                 y: e.pageY - $window.scrollTop()
             },
-            allObjects;
+            allObjects = that.getElementsFromPoint(mousePosition);
+
+        that.verticalScrollHelper && that.verticalScrollHelper.updateScrollable(allObjects, mousePosition);
```

Comment:
Add "_" prefix to verticalScrollHelper  and horizontalScrollHelper because they are not public